src/backend/distributed/commands/function.c

```diff
@@ -1366,6 +1366,11 @@ PreprocessCreateFunctionStmt(Node *node, const char *queryString,
 List *
 PostprocessCreateFunctionStmt(Node *node, const char *queryString)
 {
+	if (!EnablePropagationWarnings)
```

Comment:
We have 2 such cases (QualifyAlterDomainStmt and PostprocessCreateFunctionStmt) in which we change behaviour because they cause postgres test failures. I need to investigate those.